go-controller/pkg/cluster/gateway_init.go

```diff
@@ -11,33 +11,37 @@ import (
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 )
 
-// bridgedGatewayNodeSetup makes the bridge's MAC address permanent, sets up
+// bridgedGatewayNodeSetup makes the bridge's MAC address permanent (if needed), sets up
 // the physical network name mappings for the bridge, and returns an ifaceID
 // created from the bridge name and the node name
-func bridgedGatewayNodeSetup(nodeName, bridgeInterface string) (string, string, error) {
+func bridgedGatewayNodeSetup(nodeName, bridgeName, bridgeInterface string, brCreated bool) (string, string, error) {
```

Comment:
@girishmg I'd rename `brCreated` to `syncBridgeMAC` or `forceBridgeMAC` or something like that, since that's what its really doing.

Reply:
Will do